taste_tester.gemspec

```diff
@@ -14,7 +14,7 @@
 
 Gem::Specification.new do |s|
   s.name = 'taste_tester'
-  s.version = '0.0.13'
+  s.version = '0.0.14'
```

Comment:
don't do this in this PR... when we're ready to release, then we can do that...

Reply:
Reverted. I am testing locally with a different version number though, so please do a sanity check before the final merge that I don't accidentally push this again.